packages/manager/src/features/Billing/BillingPanels/ContactInfoPanel/ContactInformation.tsx

```diff
@@ -208,18 +214,24 @@ const ContactInformation: React.FC<CombinedProps> = (props) => {
                 taxId !== undefined && taxId !== null && taxId !== '',
             })}
           >
-            <div className={classes.section} data-qa-contact-email>
-              <div className={classes.wordWrap}>{email}</div>
-            </div>
-
+            <Typography
+              className={`${classes.section} ${classes.wordWrap}`}
+              data-qa-contact-email
+            >
+              {email}
+            </Typography>
             {phone && (
-              <div className={classes.section} data-qa-contact-phone>
+              <Typography className={classes.section} data-qa-contact-phone>
                 {phone}
-              </div>
+              </Typography>
             )}
-
             {taxId && (
-              <div className={classes.section}>{'Tax ID ' + taxId}</div>
+              <Typography
+                className={classes.section}
+                style={{ marginTop: 'auto' }}
+              >
+                <span className={classes.bold}>Tax ID</span> {taxId}
```

Comment:
We don't need the bold class if we use the strong tag
```suggestion
                <strong>Tax ID</strong> {taxId}
```